### ISSUE TITLE
I just reinstalled the updated version and I am getting the same issue with the themes.  I now have the "PNG_EXPORTS" In the main folder.  

### ISSUE THREAD

**Issue by codyconley** [amanatron/Maestoso#3]

**Comment by amanatron** [comment]
That's odd. Have you installed everything from the requirements.txt and can you show me the error? Is it the exact same one as before? i.e, does it mention the exact same themes that failed to load? 

If it's just one or two themes, then you're most definitely missing the mplcyberpunk library.

**Comment by codyconley** [comment]
> That's odd. Have you installed everything from the requirements.txt and can you show me the error? Is it the exact same one as before? i.e, does it mention the exact same themes that failed to load?
> 
> If it's just one or two themes, then you're most definitely missing the mplcyberpunk library.

Yes, Everything is installed and It is the identical error to the previous issue.

**Comment by amanatron** [comment]
Okay let me try recreating your issue. You're using a windows machine, right?
